rspace/src/test/scala/coop/rchain/rspace/StorageActionsTests.scala

```diff
@@ -1387,6 +1387,55 @@ trait StorageActionsTests
     check(prop)
   }
 
+  "produce and consume" should "store channel hashes" in withTestSpace { space =>
```

Comment:
wouldn't it make sense to extend existing tests (eg. `consume twice then createCheckpoint`)? they have the benefit of being generative.

Reply:
I initially implemented these checks as part of another test, but the result wasn't look well - a mess of checks that verify non-related functionality. To make it more readable and follow single responsibility principle decided to create a separate test.